lib/nerves_hub_link/configurator.ex

```diff
@@ -156,6 +156,8 @@ defmodule NervesHubLink.Configurator do
       |> Map.put("fwup_version", fwup_version())
       |> Map.put("device_api_version", @device_api_version)
       |> Map.put("console_version", @console_version)
+      |> Map.put("library", "nerves_hub_link")
+      |> Map.put("library_version", to_string(Application.spec(:nerves_hub_link, :vsn)))
```

Comment:
```suggestion
      |> Map.put("client", "nerves_hub_link")
      |> Map.put("client_version", to_string(Application.spec(:nerves_hub_link, :vsn)))
```
It might be more applicable to call this `client` instead of library?

Reply:
Love this.